README.md

```diff
@@ -1,2 +1,29 @@
-# GCP_RaspberryPi
-Python Code to Push images captured by Raspberry Pi to Google Cloud Storage
+# Connection between Google Cloud Storage and Raspberry Pi 3B
+
+## Target
+- Raspberry pi is most commonly used device in IoT Platform 
```

Comment:
'Raspberry Pi' has got some branding guideline - the P in Pi should be capitalized.
read more [here](https://www.raspberrypi.org/trademark-rules/)